scripts/globals/dealer_moogle.lua

```diff
@@ -9,6 +9,13 @@ require("scripts/globals/zone")
 xi = xi or {}
 xi.dealerMoogle = xi.dealerMoogle or {}
 
+local debug =
+{
+    ENABLED     = false,    -- will disable ki consumption and print instead
+    SHOWITEM    = true,     -- will display acquisition message to player if debug.ENABLED
+    TO_PLAYER   = true,     -- will print debug info to player if debug.ENABLED
```

Comment:
Not oposed to debug messages being there for the tme being.

However, could you disable them by default?

Reply:
Functionally `SHOWITEM` and `TO_PLAYER` are completely ignored unless `ENABLED` is true. That said, happy to flag them all as false by default.